UI/api-interface.cpp

```diff
@@ -620,6 +620,12 @@ struct OBSStudioAPI : obs_frontend_callbacks {
 		return bstrdup(recordOutputPath);
 	}
 
+	char *obs_frontend_get_translated_string(const char *string) override
```

Comment:
Is `obs_frontend_get_translated_string` the right name here? Should it be `obs_frontend_get_locale_string` like `obs_module_get_locale_string`?

Reply:
Done